src/index.js

```diff
@@ -0,0 +1,11 @@
+import _ from 'lodash';
+
+function component() {
+  const element = document.createElement('div');
+
+  element.innerHTML = _.join(['Hello', 'webpack'], ' ');
+
+  return element;
+}
+
+document.body.appendChild(component());
```

Comment:
Is there a reason you chose this method? If not, I suggest creating a div within your index.html file and adding an id.